packages/traceloop-sdk/package.json

```diff
@@ -35,15 +35,18 @@
     "@opentelemetry/exporter-trace-otlp-proto": "^0.44.0",
     "@opentelemetry/sdk-node": "^0.44.0",
     "@traceloop/instrumentation-openai": "^0.0.24",
+    "@traceloop/instrumentation-llamaindex": "^0.0.24",
     "@types/nunjucks": "^3.2.5",
     "fetch-retry": "^5.0.6",
     "nunjucks": "^3.2.4",
     "openai": ">=4.0.0",
+    "llamaindex": ">=0.0.40",
```

Comment:
why does it need it as a direct dependency?

Reply:
Should I remove openai as well?

Reply:
Yeah, weird how it got there. Should be peer only